src/mesh/mesh.cpp

```diff
@@ -118,9 +118,9 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Properties_t &properti
   }
 
   // check number of grid cells in root level of mesh from input file.
-  if (mesh_size.nx1 < 4) {
+  if (mesh_size.nx1 < 1) {
```

Comment:
@Yurlungur is it OK if I do this? I wanted to be able to run one-zone problems

Reply:
I'm not sure what the full implications of this are. I think it should be fine. Others may also know better. @jdolence @jmstone ?

Reply:
@Yurlungur @jdolence I'm going to go ahead and leave this in since it seems consistent with the following
```c++
if (mesh_size.nx2 < 1)
if (mesh_size.nx3 < 1)
```
unless I hear otherwise